documentation/rfc/distributed.md

```diff
@@ -0,0 +1,36 @@
+Scenario Distributed
+====================
+
+
+This scenario allow us to call multiple engines for computing fallbacks. These engines can be:
+
+  - kraken
+  - geovelo
+  - here
+  - valhalla
+
+The orchestration is a bit complex, as such this documentation will try to explain what is done, and it what order.
+Each fallback mode can use a specific engine that is configured at the instance level.
```

Comment:
instance ? coverage ?

Reply:
In the jormungandr routing, aka urls it is call coverage, in some endpoint it is called region, and the jormungandr source code, the object is called `Instance`...

Region is almost never used and is just kept for retro compatibility purpose at the API level.
But we freely use instance and coverage to designate it.